### ISSUE TITLE
Uncaught exception - load.c(237) : Failed to load library : lime.ndll

### ISSUE THREAD

**Issue by CrazyFlasher** [openfl/lime#1357]
If I use local haxelib `haxelib newrepo`
I get this compile error
```
Called from openfl/_internal/renderer/context3D/Context3DGraphics.hx line 34
Called from openfl/display/BitmapData.hx line 130
Called from a C function
Called from openfl/display/BitmapData.hx line 294
Called from lime/graphics/Image.hx line 548
Called from lime/_internal/graphics/ImageDataUtil.hx line 526
Called from a C function
Called from /usr/share/haxe/std/neko/Lib.hx line 42
Uncaught exception - load.c(237) : Failed to load library : lime.ndll
Error: Command failed with error 1
```

**Comment by player-03** [comment]
I know I'm three years late, but do you have any recollection of what you were compiling? ~~Also I'm pretty sure this was a runtime error; there should be no reason to initialize `Context3DGraphics` at compile time.~~ Edit: no, it would initialize static fields even at compile time, as long as the class was imported. And lots of things get imported by accident.

~~That being said, I have run into a similar problem caused by that line of `Context3DGraphics`. Regardless of the details of this bug, it should be fixed by openfl/openfl#2543.~~ Edit: I rescinded that PR, so it won't solve the problem.

**Comment by player-03** [comment]
I think my previous ideas were wrong. But good news: if this is related to that local Haxelib repo, #1992 should hopefully fix it.

**Comment by tobil4sk** [comment]
I don't think that PR would solve this issue, since it will only affect the behaviour library resolution within `run.n` so it shouldn't have an effect outside of that. I think it's most likely that the error comes form the `swf` library since that runs at build time and uses openfl.
